interface/wx/graphics.h

```diff
@@ -633,46 +640,54 @@ class wxGraphicsContext : public wxGraphicsObject
         of gradient @a stops can be specified.
 
         The version taking wxGraphicsGradientStops is new in wxWidgets 2.9.1.
+
+        The ability to apply a transformation matrix to the gradient was added in 3.1.3
```

Comment:
I think we mostly use wording like `@a matrix parameter is only available since wxWidgets 3.1.3`. I'm not really sure, but I'd just like to make it consistent with the existing documentation.